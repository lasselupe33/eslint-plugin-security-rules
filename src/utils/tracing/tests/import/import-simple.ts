import { a, b, c, myFunc, MyClass } from "./exports/export-simple";

const instance = new MyClass("eVal");

const start = `${a}-${b}-${c}-${myFunc("dVal")}-${instance.a}`;
