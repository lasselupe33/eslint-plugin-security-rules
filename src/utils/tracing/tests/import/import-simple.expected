aVal-bVal-cVal-myFunc-dVal-classValeVal